F1/F1Tests/StandingsViewModelTests.swift

```diff
@@ -1,9 +1,113 @@
+////
+////  StandingsViewModelTests.swift
+////  F1Tests
+////
+////  Created by Kaitlyn Sookdhew on 2024/05/05.
+////
 //
-//  StandingsViewModelTests.swift
-//  F1Tests
+//import XCTest
+//@testable import F1
 //
-//  Created by Kaitlyn Sookdhew on 2024/05/05.
+//class MockStandingsNavigationDelegate: StandingsNavigationDelegate {
```

Comment:
Please don't commit empty/commented out files to dev

Reply:
Sorry 😄